.eslintrc.js

```diff
@@ -20,37 +20,37 @@ module.exports = {
     // 'error' or 2 - turn the rule on as an error (exit code will be 1)
     rules: {
         // Best Practices
-        'block-scoped-var': 2,
-        complexity: [2, { max: 40 }],
-        'consistent-return': 2,
-        curly: 2,
-        'dot-location': [2, 'property'],
-        'dot-notation': [2, { allowPattern: '^[a-z]+(_[a-z]+)+$' }],
-        eqeqeq: 0,
-        'no-alert': 2,
-        'no-multi-spaces': 2,
-        'no-param-reassign': 0,
-        'no-redeclare': 2,
-        'no-unused-expressions': [2, { allowShortCircuit: true, allowTernary: true }],
-        'vars-on-top': 0,
-        yoda: [2, 'never', { exceptRange: true }],
-        'no-console': 1, //For debugging, but throw warnings so logs are not commited.
-        'no-useless-escape': 0,
+        'block-scoped-var': 'error',
+        complexity: ['error', { max: 40 }],
+        'consistent-return': 'error',
+        curly: 'error',
+        'dot-location': ['error', 'property'],
+        'dot-notation': ['error', { allowPattern: '^[a-z]+(_[a-z]+)+$' }],
+        eqeqeq: 'error', 
```

Comment:
// certain examples are nice to use non-literal eq.
        // typeof foo == 'undefined'
        // 'hello' != 'world'
        // 0 == 0
        // true == true
        // foo == null
        // "smart" is a good balance option vs error.
[eslint example](https://eslint.org/docs/latest/rules/eqeqeq#rule-details)